main.js

```diff
@@ -10,4 +10,193 @@ function closeMenu() {
 }
 
 hamburger.addEventListener('click', showMenu);
-close.addEventListener('click', closeMenu);
\ No newline at end of file
+close.addEventListener('click', closeMenu);
+
+// lets make popup daynamically,
+
+// object,
+
+const projects = [
+  {
+    fullName: 'Multi',
+    technologiesOne: 'html',
+    technologiesTwo: 'Bootstrap',
+    technologiesThree: 'Ruby on rails',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+
+    linkToLive: 'See Live',
+    linkToSource: 'See Source',
+  },
+  {
+    fullName: ' Post',
+    technologiesOne: 'html',
+    technologiesTwo: 'Bootstrap',
+    technologiesThree: 'Ruby on rails',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+
+    linkToLive: 'See Live',
+    linkToSource: 'See Source',
+  },
+  {
+    fullName: ' Stories',
+    technologiesOne: 'html',
+    technologiesTwo: 'Bootstrap',
+    technologiesThree: 'Ruby on rails',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+
+    linkToLive: 'See Live',
+    linkToSource: 'See Source',
+  },
+
+];
+
```

Comment:
- [ ] Kindly make sure to add the same number of the project objects equals to the total number of the project rendered on the page.  At this moment we have only 3 project full details objects but on the page, we have more that 5 projects. Kindly fix that and make sure to provide unique info for each project.